flink-contrib/docker-flink/docker-entrypoint.sh

```diff
@@ -36,9 +39,9 @@ elif [ "$1" == "taskmanager" ]; then
     echo "Starting Task Manager"
     echo "config file: " && grep '^[^\n#]' $FLINK_HOME/conf/flink-conf.yaml
     $FLINK_HOME/bin/taskmanager.sh start
+
+  # prevent script to exit
+  tail -f /dev/null
 else
     $@
```

Comment:
When this image is used as a one-off container, e.g. I want to run `bin/flink list`, then this line would prevent the one-off container to exit correctly - instead it would hang forever.